srcpkgs/yaru/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'yaru'
+pkgname=yaru
+version=r28
+revision=1
+archs=noarch
+wrksrc="yaru-$version"
+build_style=meson
+makedepends="glib-devel gtk+3-devel meson sassc"
```

Comment:
glib-devel is a package for tooling while building packages (like glib-mkenums) and it should
be in hostmakedepends=

gtk+3-devel is the library so it should stay in makedepends=

build_style=meson already adds a dependency on meson so you don't need to define it again

sassc is an executable and should go to hostmakedepends=

Please read up on the different types of depends https://github.com/void-linux/void-packages/blob/master/Manual.md#explain_depends